templates/unitarity.cpp.in

```diff
@@ -0,0 +1,80 @@
+// ====================================================================
+// This file is part of FlexibleSUSY.
+//
+// FlexibleSUSY is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published
+// by the Free Software Foundation, either version 3 of the License,
+// or (at your option) any later version.
+//
+// FlexibleSUSY is distributed in the hope that it will be useful, but
+// WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with FlexibleSUSY.  If not, see
+// <http://www.gnu.org/licenses/>.
+// ====================================================================
+
+
+/**
+ * @file @ModelName@_unitarity.cpp
+ *
+ * This file was generated with FlexibleSUSY @FlexibleSUSYVersion@ and SARAH @SARAHVersion@ .
+ */
+
+#include "@ModelName@_unitarity.hpp"
+#include "@ModelName@_mass_eigenstates.hpp"
+#include "cxx_qft/@ModelName@_fields.hpp"
+
+#include "sum.hpp"
+#include "wrappers.hpp"
+
+#include <Eigen/Eigenvalues>
+
+namespace flexiblesusy {
+namespace @ModelName@_unitarity {
+
+static constexpr int size = @scatteringPairsLength@;
+
+namespace {
+
+inline
+double Sqrt2(int i, int j) {
```

Comment:
Would it make sense to rename this function? Maybe to `Sqrt2IfEq` or so?

Reply:
I could do that. But there's actually a logic behind it. That's how Mark called it. And this whole thing is just a wrapper around his calculation. But I don't have a strong opinion on this issue. So if you like I can rename it.

Reply:
Ah, good point. It would make sense to leave the name for comparison with SARAH. Then I'd propose to leave the name as it is.